src/Select.js

```diff
@@ -87,12 +80,30 @@ export class Select extends Component {
         })
     }
 
+    handleMeasurementStart = () => {
+        this.updateMenuPosition()
+
+        window.addEventListener('resize', this.updateMenuPosition)
+        window.addEventListener('scroll', this.updateMenuPosition)
```

Comment:
I don't think we need the scroll event listener.

The Backdrop stops the user from scrolling, so it will never fire.

![modal-scroll](https://user-images.githubusercontent.com/185449/69173912-0aa9e280-0b01-11ea-920d-ea1022cbd33f.gif)

Reply:
(When I wiggle the mouse cursor up and down I'm using the scroll wheel.)

Reply:
Maybe it's different on Mac?

Reply:
Strange. At first I thought maybe I was working on an outdated version of next. But I've rebased this branch on next and can still scroll with the Select open, both of FF and Chrome. I can scroll with the touchpad and with the arrow keys with the menu open.

Reply:
We could either leave the scroll listeners for the Select, or if we consider it necessary for the Backdrop to stop scrolling, make sure that it stops scrolling on mac as well.

Reply:
I can also scroll, same as @ismay. What I've been reading so far about preventing scrolling effectively, is to put some blocking styles on the document body. I'd be in favour of just keeping the scroll and resize listeners for 4.0.0

Reply:
Interesting! I need to find someone with a Mac and see this in action.